server/pages/index.jade

```diff
@@ -110,25 +110,45 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
-		if user
+		if supportUser
+			script.
+				window.supportUser = {
+					readyScripts: []
+				};
+		if user && !supportUser
 			script(type='text/javascript')!='var currentUser = ' + sanitize.jsonStringifyForHtml( user )
 		if app
 			script(type='text/javascript')!='var app = ' + sanitize.jsonStringifyForHtml( app )
-		if initialReduxState
+		if initialReduxState && !supportUser
```

Comment:
If support user is active, the API data injected here will belong to the original user and not the supported user, which we'd like to ignore while support user is active. We'll need to fetch the data and re-render after the support user token has been applied anyway, which doesn't happen until later in the Calypso boot process. Could we instead initialize it with something like `initialReduxState = {}` when support user is active?